packages/hint-compat-api/package.json

```
@@ -0,0 +1,71 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "description": "hint to validate if the HTML, CSS and JS APIs of the project are deprecated or not broadly supported",
+  "devDependencies": {
+    "@hint/utils-tests-helpers": "^1.0.0",
+    "@types/debug": "0.0.31",
+    "@types/node": "8.0.14",
+    "ava": "^0.25.0",
+    "cpx": "^1.5.0",
+    "eslint": "^5.7.0",
+    "eslint-plugin-markdown": "^1.0.0-beta.7",
+    "eslint-plugin-typescript": "^0.12.0",
+    "hint": "^3.4.10",
+    "markdownlint-cli": "^0.13.0",
+    "mdn-browser-compat-data": "0.0.54",
+    "npm-link-check": "^2.0.0",
+    "npm-run-all": "^4.1.2",
+    "nyc": "^13.1.0",
+    "rimraf": "^2.6.2",
+    "typescript": "^3.0.3",
+    "typescript-eslint-parser": "^20.0.0"
+  },
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "dist/src"
+  ],
+  "homepage": "https://webhint.io/",
+  "keywords": [
+    "hint",
+    "hint",
+    "compat-api",
+    "compat-api-hint"
+  ],
+  "license": "Apache-2.0",
+  "main": "dist/src/index.js",
+  "name": "@hint/hint-compat-api",
+  "nyc": {
+    "extends": "../../.nycrc"
+  },
+  "peerDependencies": {
+    "hint": "^3.4.10"
+  },
+  "private": true,
+  "repository": "webhintio/hint",
+  "scripts": {
+    "build": "npm run clean && npm-run-all build:*",
+    "build-release": "npm run clean && npm run build:assets && tsc -b --inlineSourceMap false --removeComments true",
```

```suggestion
    "build-release": "npm run clean && npm run build:assets && tsc --inlineSourceMap false --removeComments true",
```

Ref: https://github.com/webhintio/hint/pull/1411